docs/gallery.rst

```diff
@@ -0,0 +1,204 @@
+    :width: 200px
+    :height: 100px
+    :alt: GaussianNB Classification Report
+    :target: api/classifier/classification_report.html#classification-report
+
+.. image:: api/classifier/images/confusion_matrix.png
+    :width: 200px
+    :height: 100px
+    :alt: Logistic Regression Confusion Matrix
+    :target: api/classifier/confusion_matrix.html#confusion-matrix
+
+.. image:: api/classifier/images/rocauc.png
+    :width: 200px
+    :height: 100px
+    :alt: ROC Curves for Logistic Regression
+    :target: api/classifier/rocauc.html#rocauc
+
+.. image:: api/classifier/images/class_balance.png
+    :width: 200px
+    :height: 100px
+    :alt: Class Balance for Random Forest Classifier
+    :target: api/classifier/class_balance.html#class-balance
+
+.. image:: api/classifier/images/class_prediction_error.png
+    :width: 200px
+    :height: 100px
+    :alt: Class Prediction Error for Random Forest Classifier
+    :target: api/classifier/class_prediction_error.html#class-prediction-error
+
+.. image:: api/classifier/images/spam_discrimination_threshold.png
+    :width: 200px
+    :height: 100px
+    :alt: Threshold Plot for Logistic Regression
+    :target: api/classifier/threshold.html#discrimination-threshold
+
+Clustering Visualizers
+----------------------
+
+.. image:: api/cluster/images/elbow.png
+    :width: 200px
+    :height: 100px
+    :alt: Distortion Score Elbow for Mini Batch Means Clustering
+    :target: api/cluster/elbow.html#elbow-method
+
+.. image:: api/cluster/images/silhouette.png
+    :width: 200px
+    :height: 100px
+    :alt: Silhoutte Plot of Mini Batch Kmeans Clustering
+    :target: api/cluster/silhouette.html#silhouette-visualizer
+
+Model Selection Visualizers
+---------------------------
+
+.. image:: api/model_selection/images/validation_curve_regressor.png
+    :width: 200px
+    :height: 100px
+    :alt: Validation Curve for Decision Tree Regresor
+    :target: api/model_selection/validation_curve.html#validation-curve
+
+.. image:: api/model_selection/images/learning_curve_classifier.png
+    :width: 200px
+    :height: 100px
+    :alt: Learning Curve for MultinomialNB
+    :target: api/model_selection/learning_curve.html#classification
+
+.. image:: api/model_selection/images/learning_curve_clusterer.png
+    :width: 200px
+    :height: 100px
+    :alt: Learning Curve for KMeans
+    :target: api/model_selection/learning_curve.html#clustering
+
+Text Modeling Visualizers
+---------------------------
+
+.. image:: api/text/images/freqdist_corpus.png
+    :width: 200px
+    :height: 100px
+    :alt: Validation Curve for Decision Tree Regresor
+    :target: api/text/freqdist.html#token-frequency-distribution
+
+.. image:: api/text/images/tsne_all_docs.png
+    :width: 200px
+    :height: 100px
+    :alt: TSNE Projection of Documents
+    :target: api/text/tsne.html#t-sne-corpus-visualization
+
+Decision Boundaries Vizualizer
+------------------------------
```

Comment:
This looks great.  The layout is simple and clean

Reply:
@ralle123  We could address a couple of the things discussed above in another PR.
1. Links back would be great
2. Sphinx gallery - we are not familiar with it but would be interested to learn more